rednotebook/help.py

```diff
@@ -168,6 +168,25 @@
 ""file:///home/""]`` becomes [Home ""file:///home/""]).
 
 
+== Entry references ==
+
+You can reference days in your journal by simply writing the date in
+YYYY-MM-DD format. Such a date will be rendered as a link which can
+be clicked in preview mode.
+
+Alternatively, you can name your references, for instance:
+
+  ``[Today 2019-02-14] was a good day``
+
+will be shown as
+
+  [Today 2019-02-14] was a good day
+
+Note: Reference links are valid only in preview mode. If you try to 
+open it in a browser, you will likely get an ``Unknown protocol``
+error.
```

Comment:
Could you please make sure that journals exported to HTML don't have these broken links? Ideally, the references would take the user to the anchor of the referenced day. If that's too complicated to implement, plain text (instead of the broken link) is also fine, I think.

Reply:
I have started to work on it (in the `markup.py` as you suggested).
The problem is that the same conversion procedure is used for both in-app preview as well as HTML export.

I think it can be easily untangled but tomorrow I will be leaving country for a few months and I am afraid this patch will have to wait until my return :(